Vagrantfile

```diff
@@ -114,11 +115,11 @@ ansible_provision = proc do |ansible|
     # Use monitor_address_block instead of monitor_interface:
     ansible.extra_vars.delete(:monitor_interface)
     ansible.extra_vars = ansible.extra_vars.merge({
-      cluster_network: "#{SUBNET}.0/16",
+      cluster_network: "#{CLUSTER_SUBNET}.0/16",
```

Comment:
@batrick I think the reason it's a partial address is because it simplifies setting static IP addresses as the host part is just appended farther down in the Vagrantfile.